articles/quickstarts/index.md

```diff
@@ -2,7 +2,7 @@
 title: Install the Microsoft Quantum Development Kit (QDK)
```

Comment:
Should change to 'get started with the Microsoft Quantum Development Kit' since it describes other ways to get the QDK without installing